t3-2015/week_six.md

```diff
@@ -0,0 +1,33 @@
+Week 6 News letter
+
+This week here at Davinci Coders, it was much of the same. Yet, at the same time, all very new… We got back our grades 
+for the first assessment, which was over making classes and methods, specifically a Credit Card class with payment methods. 
+The test covered the entire process from making a git repo, to setting up our environment. On Monday, we got over our grades 
+and then went over step by step how to do the test.
+
+Wednesday we introduced Rails this week after covering more of test driven development, which is a very interesting and 
+frustrating way to write code. You write a failing test, then make it pass. Doing this for every small step of the process. 
+Now this is using Rspec which uses a different syntax than Ruby. This has been an interesting obstacle to overcome, as we are 
+all just getting use to the syntax of Ruby, which is quite nice if I might add. We talked about how the World Wide Web works, 
+which was a really interesting lecture that talked about the history of the web and how it got to where it was now. Oh, and Jason
+showed us a few terminal commands that deal with the internet, such as traceroute which traces the route an ip address takes, 
+really clarifying what latency is and why. Also I should note that we finally got the green light to use GItX which is a GUI for git and github. 
+
+Rails!!! Wednesday after break we started rails, learning how to very quickly generate a new rails app, while also 
+briefly covering the rails server and setting up the routes. But mostly we were left to our own devices when it came 
```

Comment:
`left to our own devices` vs `the homework was more challenging this time around because...`